content/overview/pricing/pricing-terminology.md

```diff
@@ -16,9 +16,11 @@ These terms, among others, define a number of products and services offered by c
 
 ### System
 
-A “System” roughly corresponds to a product or project team, including all the individual spaces, apps, and services necessary to realize their service, across as many environments as needed.
+A “system” roughly corresponds to a product or project team, including all the individual spaces, apps, and services necessary to deliver their product to users.
 
-A system maps to an [“org” in Cloud Foundry](http://docs.cloudfoundry.org/concepts/roles.html#orgs).
+A system maps to an [“org” in cloud.gov and Cloud Foundry](http://docs.cloudfoundry.org/concepts/roles.html#orgs).
+
+*Recommendation:* If your product needs Authority to Operate (ATO) from your agency, we suggest that your cloud.gov “system” should align with your ATO system boundary. (In other words, your cloud.gov org shouldn't contain two or more products that each need their own ATO, because that would make your compliance process more difficult.) If you have a very complex product with multiple sub-systems, you may choose to build that with multiple cloud.gov orgs/“systems”. For formal guidance on system boundaries, see [NIST Special Publication 800-18, section 2](http://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-18r1.pdf).
```

Comment:
I think that this is too narrow a definition. The system boundary for ATO purposes likely won't include all of their staging and development spaces, only their production spaces. But we want them to feel free to include their staging and development spaces in their org! (This is analogous to the cloud.gov SSP not including information about the staging environment, because it's not inside the ATO boundary.)